### PR TITLE
Add `nbt` command

### PR DESCRIPTION
**Pull request by Yurihaia**

### CODE REVIEW

File: cogbot/extensions/mcnbtdoc.py [hunk 1]
```diff
@@ -0,0 +1,915 @@
...
+from discord import Embed
+from discord import Color
+
+from datetime import datetime, timedelta
+
+from cogbot import checks
+from cogbot.cog_bot import CogBot
+
+import math
+import functools
```
Comment (Arcensoth) [inline cogbot/extensions/mcnbtdoc.py]: Doesn't look like `functools` is being used anywhere?

File: cogbot/extensions/mcnbtdoc.py [hunk 2]
```diff
@@ -0,0 +1,915 @@
...
+        if ace.should_scroll():
+            ace.is_scrolling = True
+            await self.bot.add_reaction(thismsg, u'◀')
+            await self.bot.add_reaction(thismsg, u'▶')
+        await self.bot.add_reaction(thismsg, u'🔼')
+        await self.bot.add_reaction(thismsg, u'🔽')
+
+    @checks.is_manager()
+    @commands.command(pass_context=True, name='nbtreload', hidden=True)
+    async def cmd_invitereload(self, ctx: Context):
```
Comment (Arcensoth) [inline cogbot/extensions/mcnbtdoc.py]: This should probably be `cmd_nbtreload` just for readability sake. (I probably forgot to update it myself in the original nbtpath extension.)

File: examples/commander_bot.json [hunk 3]
```diff
@@ -84,8 +84,12 @@
...
       "database": "https://raw.githubusercontent.com/MinecraftCommands/commanderbot/master/config/invite.json"
     },
-    "cogbot.extensions.mcnbtpaths": {
-      "database": "https://raw.githubusercontent.com/MrYurihi/mc-nbt-paths/master/output.json"
+    "cogbot.extensions.mcnbtdoc": {
+      "version_database": "https://raw.githubusercontent.com/MrYurihi/mc-nbtdoc/{}-gen/build/generated.json",
+      "database": "https://raw.githubusercontent.com/MrYurihi/mc-nbtdoc/generated/build/generated.json",
+      "ac_limit": 600,
+      "search_limit": 20,
+      "field_limit": 10
```
Comment (Arcensoth) [inline examples/commander_bot.json:92]: Can you explain what the options `ac_limit`, `search_limit`, and `field_limit` control? I can infer usage from the code but having your explanation helps me follow along.

Comment (Yurihaia) [reply examples/commander_bot.json:92]: ac limit is the time until an active embed deactivates (in seconds)
search limit is the limit to the entries on a `-ns` query
field limit is the limit to the number of fields in an embed (and in the `-s` query)

I would definitely set `field_limit` to no more than 15 just for readability,
Also, the time counter for `ac_limit` get reset when someone uses the embed, so 5 minutes could also work

**Review by Arcensoth — APPROVED** [review]

**Pull request merged**